### DISCUSSION TITLE
Mask sound wave with region

### DISCUSSION THREAD

**Discussion by QQizi** [katspaugh/wavesurfer.js#3926] [Q&A]
<img width="695" alt="Capture d’écran 2024-11-04 à 08 49 57" src="https://github.com/user-attachments/assets/6751bd4a-36fd-41aa-8ff1-6dacb78b2560">

hello, is there any way with region to do something like the image. Coloring the sound wave but not a full div ?

**Comment by katspaugh** [comment]
You can do it via a custom render function, see https://github.com/katspaugh/wavesurfer.js/discussions/3839

Alternatively, via CSS: https://github.com/katspaugh/wavesurfer.js/discussions/3794
